tests/integrations/test_zk_evm.py

```diff
@@ -93,19 +82,16 @@ async def test_execute(self, zk_evm: StarknetContract, params: dict, request):
 
     @pytest_asyncio.fixture(scope="module")
     async def erc_20(self, zk_evm: StarknetContract) -> dict:
-        solidity_output_path = Path("tests") / "solidity_files" / "output"
-        abi = json.load(open(solidity_output_path / "ERC20.abi"))
-        bytecode = (solidity_output_path / "ERC20.bin").read_text()
-        w3 = Web3()
-        ERC20 = w3.eth.contract(abi=abi, bytecode=bytecode)
-        deploy_bytecode = self.hex_string_to_bytes_array(
-            ERC20.constructor("name", "symbol", 18).data_in_transaction[2:]
```

Comment:
do you know why we were slicing the array and not anymore?

Reply:
I've updated the `hex_string_to_bytes_array` utils to make it handle leading `0x`, see `tests/utils/utils.py`